### ISSUE TITLE
[Feature Request]

### ISSUE THREAD

**Issue by JayDigg** [drinfernoo/plugin.video.openmeta#8]
It would be great if we can contribute  any and all Players we have gathered or created. Maybe a way to submit players to add to the repo so the zipball.zip can be included in the addon and updated here on this repo. Any thoughts on this idea?

**Comment by rickdoesxmas** [comment]
put em on my gh

**Comment by drinfernoo** [comment]
I don't believe we intend on including the URL for players in the add-on, but you can submit files here if you'd like to see them included. I'll also be updating this repo soon with a guide on how to make players, though it's not quite ready and probably won't be comprehensive, due to the nature of different add-ons handling URIs in sometimes very different ways.

**Comment by drinfernoo** [comment]
I've updated the wiki on this repo with a mostly comprehensive guide, with examples, of how to write a functioning player file for OpenMeta.

**Comment by mythical123** [comment]
You cant direct into all movies or all TV shows to use the full function of the EIM.  They are there if u go into the addon but not if you want to create a shortcut in a build for movies to open up the extended info mod.

**Comment by drinfernoo** [comment]
@mythical123 I answered and closed your other issue. Create a shortcut to the Video add-on version of OpenInfo if you want to do that.
